service/controller/machinepoolexp/handler/migration/testdata/case-0-Simple-MachinePool_AzureMachinePool.golden

```diff
@@ -0,0 +1,52 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
Expected AzureMachinePool CR after migration handler has initially created it, with Metadata and Spec set.